src/ert/gui/suggestor/_colors.py

```diff
@@ -0,0 +1,15 @@
+BLUE_BACKGROUND = "#d5eaf4"
+BLUE_TEXT = "#045e8d"
+RED_BACKGROUND = "#ffd2d1"
+RED_TEXT = "#90042c"
+YELLOW_BACKGROUND = "#ffe7d6"
+YELLOW_TEXT = "#7f4600"
+
+__all__ = [
+    "BLUE_BACKGROUND",
+    "BLUE_TEXT",
+    "RED_BACKGROUND",
+    "RED_TEXT",
+    "YELLOW_BACKGROUND",
+    "YELLOW_TEXT",
+]
```

Comment:
Will these be reused somewhere else? Otherwise we could have incorporated them into the `_suggestor_message` class?